src/main.js

```diff
@@ -13,43 +13,83 @@ var currentCover = Cover
 // Create your event handlers and other functions here 👇
 
 
-// We've provided one function to get you started
-// function getRandomIndex(array) {
-//   return Math.floor(Math.random() * array.length);
-// }
-
-// class RomCom {
-//   constructor() {
-//
-//   }
-// }
-
-
 
+var homeView = document.querySelector(".home-view");
+var showFormView = document.querySelector(".form-view");
+var savedView= document.querySelector(".saved-view");
 var randomCoverButton = document.querySelector(".random-cover-button");
+var homeButton = document.querySelector(".home-button");
+var saveCoverButton = document.querySelector(".save-cover-button");
 var currentTitle = document.querySelector('.cover-title');
-var currentImageSrc = document.querySelector('.cover-image').src = getRandomBookInfo(covers);
+var currentImage = document.querySelector('.cover-image');
 var currentTagLine1 = document.querySelector('.tagline-1');
 var currentTagLine2 = document.querySelector('.tagline-2');
 
 
-randomCoverButton.addEventListener("click", changeButton);
-function changeButton() {
-currentTitle.innerText = getRandomBookInfo(titles);
-//currentImageSrc.innerText = getRandomBookInfo(covers);
-currentImageSrc = document.querySelector('.cover-image').src = getRandomBookInfo(covers);
 
-currentTagLine1.innerText = getRandomBookInfo(descriptors);
-currentTagLine2.innerText = getRandomBookInfo(descriptors);
+window.addEventListener("click", clickHandler);
+window.addEventListener("load", randomCover);
+
+
+//redo for hide element
+// function hideElement(element) {
+//   document.querySelector(`.${element}`).classList.add("hidden");
+// }
+
+
+function clickHandler(event) {
+  if (event.target.classList.contains("make-new-button")){
+    makeNewCover()
+  } else if (event.target.classList.contains("random-cover-button")){
+    randomCover()
+  } else if (event.target.classList.contains("view-saved-button")){
+    viewSavedCovers()
+  } else if (event.target.classList.contains("home-button")){
+    viewHome()
+  }
+}
+
+
+function makeNewCover() {
+  //hideHomeView.innerHTML += ".view home-view hidden";
+  //showFormView.innerHTML = ""
+  homeView.classList.add("hidden");
+  randomCoverButton.classList.add("hidden");
+  saveCoverButton.classList.add("hidden");
+  //hideElement("home-view");......referenced in line33-34(hide elementfunction)
+  showFormView.classList.remove("hidden");
+  homeButton.classList.remove("hidden");
+}
+
+function viewSavedCovers() {
+  homeView.classList.add("hidden");
+  showFormView.classList.add("hidden");
+  savedView.classList.remove("hidden");
+  randomCoverButton.classList.add("hidden");
+  saveCoverButton.classList.add("hidden");
+  homeButton.classList.remove("hidden");
```

Comment:
These are great functions and work. Since there are many that are similar may want to use a class in the future.